server/templates/mainPages/index.html

```diff
@@ -3,25 +3,91 @@
+    in decreasing their GHG emissions. The recommendations list can also be filtered by societal co-benefits created by
+    particular actions.</p>
+  <p>*Disclaimer* The data displayed is an estimation intended to help user in the initiating their CAP process. The
+    information provided by the toolkit is NOT a comprehensive emissions inventory for the selected area.</p>
+  <br>
+  <h6>Instruction: Recommendation's Page</h6>
+  <p>1. Click on the check boxes to decide what types of data you would like to see. Select from the metrics and/or
+    sectors of emissions.</p>
+  <p>2. Click on the select all options to look at the full list of recommendations.</p>
+  <br>
+  <h4>About Our Team</h4>
+  <p>Our team is a collaboration of students from Western Colorado University and Colorado School of Mines.</p>
+  <p>Valeda Scriber</p>
+  <!-- <div id="center"><img src="../static/img/valeda.jpg" alt="Valeda" width="200"></div> -->
+  <p>After 15 years as an aerospace engineer and program manager building, testing, and launching satellites, Valeda
+    decided to shift her career focus to contributing to the ever-escalating issue of climate change. To expand her
+    knowledge of environmental issues she spent two years travelling and volunteering internationally. Eventually she
+    enrolled at Western Colorado University to join their Master of Environmental Management program with a focus in
+    Sustainable and Resilient Communities. She assisted Eagle County with emissions forecasting of climate actions and
+    updated Park City’s emissions inventory as a part of the Mountain Town 2030 (MT2030) cohort. These efforts
+    highlighted the cost, effort duration of a standard GHG emissions inventory and thus the concept of the Tiny Feet
+    CAP toolkit emerged.</p>
+  <br>
+  <p>Eric Swanson</p>
+  <!-- <div id="center"><img src="../static/img/eric.jpg" alt="Eric" width="200"></div> -->
+  <p>Eric is a non-traditional student studying Computer Science at Western Colorado University who has a bachelors in
+    Media and Communications.</p>
+  <br>
+  <p>Gillian Yost</p>
+  <!-- <div id="center"><img src="../static/img/gillian.png" alt="Gillian" width="200"></div> -->
+  <p>Gillian Yost is currently a senior at Colorado School of Mines where she is majoring in Electrical Engineering with
+    double minors in Computer Science and Public Affairs. In the last two years of her degree, Gillian shifted her focus
+    to more of the software portion of Electrical Engineering and as a result, next year in Fall 2021, Gillian will be
+    pursuing a Master of Science in Software Engineering at University of Texas in Austin. </p>
+  <div class="row">
+    <div class="column">
+      <div id="center">
+        <p>Valeda and Eric...</p>
+      </div>
       <div id="center"><img src="../static/img/madJack.png" alt="Western Logo" width="200"></div>
```

Comment:
A better way to link to static files inside this flask microframework might be to use jinja url_for function so that if the directory structure or file name are changed, the image can still be linked. Example: <img src="{{ url_for('static', filename='img/madJack.png') }}" style="width:42px;height:42px;">